Makefile.include

```diff
@@ -207,6 +207,9 @@ INCLUDES += -I$(RIOTBASE)/core/include -I$(RIOTBASE)/drivers/include -I$(RIOTBAS
 INCLUDES += -I$(RIOTCPU)/$(CPU)/include
 INCLUDES += -I$(RIOTBOARD)/$(BOARD)/include
 
+# process provided features
+-include $(RIOTBOARD)/$(BOARD)/Makefile.features
```

Comment:
(unrelated) seeing this here now: couldn't we just add `-include $(RIOTCPU)/$(CPU)/Makefile.features` in the line below and drop the cpu specific includes from all the board's Makefile.features? If yes I will provide a follow-up PR once this is merged...

Reply:
Nope, as for "global" targets, this is (unfortunately) not the case.